src/main/java/edu/msudenver/bluejext/qualitytools/QualityToolsExtension.java

```diff
@@ -91,3 +99,42 @@ public URL getURL() {
         }
     }
 }
+
+/**
+ * Create a Preference Panel located in the BlueJ application
```

Comment:
This comment seems to indicate that this class determines the location of the panel.  That doesn't seem to be the actual way this is working.